packages/zendesk-adapter/src/filters/handle_identical_attachment_conflicts.ts

```diff
@@ -53,14 +78,20 @@ const filterCreator: FilterCreator = ({ config }) => ({
           return []
         }
         // all files have the same hash, so we can remove the duplicates and keep only one
+        duplicateElemIds.add(attachments[0].elemID.getFullName())
         return attachments.slice(1).map(att => att.value.id)
       }))
-    log.debug(`going to remove duplicates for attachments: ${duplicateElemIds.join(',')}`)
-    log.debug(`ids removed: ${Array.from(idsToRemove).join(',')}`)
+    log.debug(`going to remove duplicates for attachments: ${Array.from(duplicateElemIds).join(',')}`)
```

Comment:
nit: for debug-level logs (and in general below-error), there's a preference for using arguments and not string templates so that we don't calculate the combined string if it won't be logged - in this case something like
```suggestion
    log.debug('going to remove duplicates for attachments: %s', Array.from(duplicateElemIds).join(','))
```